Tabloid/Controllers/TagController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+
+namespace Tabloid.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+
+    public class TagController : ControllerBase
```

Comment:
Should this controller be auth protected? do we need any other auth checks on the route?

Reply:
That's a good question. In my mind I read this section of the issue ticket "NOTE: For the time being it is acceptable to treat all users as admin users. There is a future story about enforcing user permissions" to mean that we would enforce the auth checks on a later ticket?

Reply:
@aliparadise Good question! So you don't need to check for authorization (ie the user doesn't need to be an admin), but the user should be authenticated (ie the user should be logged in) in order to see the tags. Let me know if you have questions about the difference there!

Reply:
I went back over that chapter and realized I was confusing myself with the use of "auth" for authenticated vs authorized. Now it's coming back to me!

Reply:
Changes made.